setup.py

```diff
@@ -134,7 +134,8 @@
         "dffml.secret": ["ini = dffml.secret.ini:INISecret"],
         # Accuracy
         "dffml.accuracy": [
-            "mse = dffml.accuracy.mse:MeanSquaredErrorAccuracy"
+            "mse = dffml.accuracy.mse:MeanSquaredErrorAccuracy",
+            "clfacc = dffml.accuracy.clfacc:ClassificationAccuracy",
```

Comment:
`clfacc` -> `clf`, we'll need to change the filename too

```suggestion
            "clf = dffml.accuracy.clf:ClassificationAccuracy",
```